core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorManagerService.js

```diff
@@ -138,6 +138,10 @@ oppia.factory('SimpleEditorManagerService', [
         data.questionList.addQuestion(QuestionObjectFactory.create(
           lastStateName, stateData.interaction, ''));
       },
+      changeQuestion: function(type,index) {
+        // This change the type of question
```

Comment:
I'm not sure about that. I wouldn't say that the interaction selector works unless someone else who implements another question type can just plug in their question type and question-specific code into a few specific well-defined places, and it "just works". They shouldn't have to fiddle with the infrastructure.